Package.swift

```diff
@@ -1,41 +1,84 @@
 // swift-tools-version:5.3
+import Darwin.C
 import PackageDescription
 
+var products: [Product] = [
+  .library(name: "Sentry", targets: ["Sentry"]),
+  .library(name: "Sentry-Dynamic", targets: ["Sentry-Dynamic"]),
+  .library(name: "SentrySwiftUI", targets: ["Sentry", "SentrySwiftUI"])
+]
+
+var targets: [Target] = [
+  .binaryTarget(
+              name: "Sentry",
+              url: "https://github.com/getsentry/sentry-cocoa/releases/download/8.52.1/Sentry.xcframework.zip",
+              checksum: "48a6c6693148a3f9096108164eb938a931b964395fcf38e169383b9e4cffcfc5" //Sentry-Static
+          ),
+  .binaryTarget(
+              name: "Sentry-Dynamic",
+              url: "https://github.com/getsentry/sentry-cocoa/releases/download/8.52.1/Sentry-Dynamic.xcframework.zip",
+              checksum: "b9d9054c65ee5ac0591c27826edddc54490a96c2a83dee25519cae0c1a593231" //Sentry-Dynamic
+          ),
+  .target ( name: "SentrySwiftUI",
```

Comment:
`m`: This file might need some formatting

```suggestion
.target(name: "SentrySwiftUI",
```